### ISSUE TITLE
Dont work with expressions like translate.google.com/#* or www.qwant.com/?q*

### ISSUE THREAD

**Issue by adelplanque** [mrdokenny/History-AutoDelete#59]
Because I want to keep translate.google.com or www.qwant.com in my history, but not translate.google.com/#* or www.qwant.com/?q*

**Comment by ts1a** [comment]
Yeah, I wanted such functionality too, but I guess it's not possible anymore. Maybe you can achieve it replacing `*` by `+` in the database file...

**Comment by grahamperrin** [comment]
#27
